spec/features/results_download_spec.rb

```diff
@@ -12,8 +17,6 @@
   it "visits results page, clicks 'Waitlist', checks if csv file is downloaded" do
```

Comment:
I mean, click the link to `Waitlist` just as we click the link to `Accepted` above. ;)